### PR TITLE
[ACS] add --ssh-key-file param with default

### PR DESCRIPTION
**Pull request by rjtsdl**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [x] Each command and parameter has a meaningful description.
- [x] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

**Comment by rjtsdl** [comment]
Fix #3410

**Comment by rjtsdl** [comment]
@troydai to review and help merge.

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py [hunk 1]
```diff
@@ -110,6 +110,12 @@ def _get_feature_in_preview_message():
 
 register_cli_argument('acs', 'disable_browser', help='Do not open browser after opening a proxy to the cluster web user interface')
 register_cli_argument('acs dcos browse', 'name', name_arg_type)
+register_cli_argument('acs dcos browse', 'ssh_key_file',
+                      required=False,
+                      help='Path to an SSH key file to use.',
+                      type=file_type,
+                      default=os.path.join('~', '.ssh', 'id_rsa'),
```
Comment (tjprescott) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py]: Is this the appropriate default on Linux and Windows?  I'd be surprised if they weren't different.

Comment (rjtsdl) [reply src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py]: we already adopted this style for other ssh keys. In the same file, you can check how we default ssh-key-value. 

Btw, in the issue, it is the desired default per the discussion.

**Pull request merged**